src/org/rust/lang/core/resolve/ref/RustModReferenceImpl.kt

```diff
@@ -0,0 +1,19 @@
+package org.rust.lang.core.resolve.ref
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiReferenceBase
+import org.rust.lang.core.psi.RustModDeclItem
+import org.rust.lang.core.psi.RustModItem
+import org.rust.lang.core.psi.util.moduleFile
+
+class RustModReferenceImpl(modDecl: RustModDeclItem) :
+        PsiReferenceBase<RustModDeclItem>(modDecl, modDecl.identifierRange, /* soft = */ false)
```

Comment:
Is there any way I can convince IDEA to indent `:` if it is on the new line? Currently, it autoformats to 

```Kotlin
class RustModReferenceImpl(modDecl: RustModDeclItem)
: PsiReferenceBase<RustModDeclItem>(modDecl, modDecl.identifierRange, /* soft = */ false)
  , RustReference {
```